addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -385,7 +402,15 @@ class EnvironmentScService extends Service {
     const { envTypeId, envTypeConfigId, projectId } = environment;
 
     // Lets find the index id, by looking at the project and then get the index id
-    const { indexId } = await projectService.mustFind(requestContext, { id: projectId, fields: ['indexId'] });
+    // The isAppStreamConfigured attribute value will be returned by project service. No other fields needed to be added
+    const { indexId, isAppStreamConfigured } = await projectService.mustFind(requestContext, {
+      id: projectId,
+      fields: ['indexId'],
+    });
```

Comment:
Outside scope of this but I just thought about this. It seems like we will need to look at other APIs like createConnectionUrl, find etc and throw similar errors